### PR TITLE
change key of item (remove endpoint)

### PR DESCRIPTION
**Pull request by ovekyc**
change item key of database, now we use **uuid** for unique

and edit response code as chaining

### CODE REVIEW

File: src/test/node/items/controllers.js [hunk 1]
```diff
@@ -99,7 +101,7 @@ test('delete an item from database', t => {
 
     const req = httpMocks.createRequest({
       method: 'DELETE',
-      url: '/items/1',
+      url: '/items/' + key,
```
Comment (fguy) [inline src/test/node/items/controllers.js]: Please use template for string concatenation. 

url: `/items/${key}`,

File: package.json [hunk 2]
```diff
@@ -44,6 +44,7 @@
     "moment": "^2.14.1",
     "morgan": "~1.7.0",
     "node-simple-logger": "0.0.1",
+    "node-uuid": "^1.4.7",
```
Comment (fguy) [inline package.json]: You probably only need UUID4. I recommend to use uuid4: https://www.npmjs.com/package/uuid4

File: src/test/node/items/controllers.js [hunk 3]
```diff
@@ -83,8 +84,9 @@ test('get a item from database', t => {
   });
 });
 test('delete an item from database', t => {
+  const key = 'item-' + uuid.v1();
```
Comment (fguy) [inline src/test/node/items/controllers.js]: template literal here. and do not use uuid1. uuid4 is completely random.

Comment (ovekyc) [reply src/test/node/items/controllers.js]: Edit it Thank you!

File: package.json (hunk 2 above)
Comment (ovekyc) [reply package.json]: We gonna use that one instead of node-uuid

File: src/test/node/items/controllers.js (hunk 1 above)
Comment (ovekyc) [reply src/test/node/items/controllers.js]: beautiful  template. i edited it

**Comment by fguy** [comment]
LGTM

**Pull request closed without merging**